### PR TITLE
F323 Initial onboarding popover

### PR DESCRIPTION
**Pull request by Matt-Innocenzo**
Help Screen implemented

### CODE REVIEW

**Comment by rolflocher** [comment]
Looks good just have to resolve those conflicts. Should the help label be under the settings button instead of on the map or something? I had to go to the storyboard to find it the first time

File: CranberryQueue/Map/MapViewController.swift [hunk 1]
```diff
@@ -424,7 +443,7 @@ class MapViewController: UIViewController, UITextFieldDelegate, MapControllerDel
     }
     
     // Join public queue from queue detail modal or player
-    func joinPublicQueue(queueId: String, name: String) {
+    @objc func joinPublicQueue(queueId: String, name: String) {
```
Comment (rolflocher) [inline CranberryQueue/Map/MapViewController.swift]: You actually mixed something up here. This was rewritten to not be objc

File: CranberryQueue/Map/MapViewController.swift [hunk 2]
```diff
@@ -103,7 +108,11 @@ class MapViewController: UIViewController, UITextFieldDelegate, MapControllerDel
         settingsIconImageView.addGestureRecognizer(settingsTap)
         settingsIconImageView.isUserInteractionEnabled = true
         
-        let joinQueueTap = UITapGestureRecognizer(target: self, action: #selector(joinDetailQueue))
+        let helpTap = UITapGestureRecognizer(target: self, action: #selector(helpTapped))
+        helpIconImageView.addGestureRecognizer(helpTap)
+        helpIconImageView.isUserInteractionEnabled = true
+        
+        let joinQueueTap = UITapGestureRecognizer(target: self, action: #selector(joinPublicQueue))
```
Comment (rolflocher) [inline CranberryQueue/Map/MapViewController.swift]: The selector should be joinDetailQueue, not joinPublicQueue

Comment (rolflocher) [reply CranberryQueue/Map/MapViewController.swift]: Not that you would expect to test this area of UX, but this actually breaks the guest path

Comment (Matt-Innocenzo) [reply CranberryQueue/Map/MapViewController.swift]: Resolved. For some reason on merging master into my branch, the functions changed but my tap gesture recognizer didn't update right so I went changing stuff to get it to run my path, didn't check guest. Just went line for line with master map view controller so it should be good.

File: CranberryQueue/Map/MapViewController.swift (hunk 1 above)
Comment (Matt-Innocenzo) [reply CranberryQueue/Map/MapViewController.swift]: should be good to go

**Pull request closed without merging**